client/Dockerfile

```diff
@@ -1,7 +1,9 @@
 # Client Build
 FROM trion/ng-cli as ng
-# Define the env config arg
+# Define args
 ARG ENVCONFIG
+ARG CERT
+ARG CERT_KEY
```

Comment:
Should we really package the certificate and key inside the docker image? This exposes the private key and requires us to rebuild the image everytime we want to change the cert/key. I think it should simply be an env variable on the vm.

Reply:
Yeah you're right. What I am thinking right now is having the keys and certificates in their respective VM and then mounting a volume to the container when we start docker compose. I think this is a valid solution and all the scripting/secrets can be removed with that. As soon as we start docker compose, the client service will mount the certificates to the ./ssl directory on the container and nginx will be able to pick them up easily.